app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,32 @@
+html lang="en" class="govuk-template"
+  head
+    title = [yield(:page_title).presence, t("service.name")].compact.join(" - ")
+
+    = csrf_meta_tags
+    = csp_meta_tag
+
+    = tag.meta name: "viewport", content: "width=device-width, initial-scale=1" 
+    = tag.meta property: "og:image", content: asset_path("images/govuk-opengraph-image.png") 
+    = tag.meta name: "theme-color", content: "#0b0c0c" 
+    = favicon_link_tag asset_path("images/favicon.ico") 
+    = favicon_link_tag asset_path("images/govuk-mask-icon.svg"), rel: "mask-icon", type: "image/svg", color: "#0b0c0c" 
+    = favicon_link_tag asset_path("images/govuk-apple-touch-icon.png"), rel: "apple-touch-icon", type: "image/png" 
+    = favicon_link_tag asset_path("images/govuk-apple-touch-icon-152x152.png"), rel: "apple-touch-icon", type: "image/png", size: "152x152" 
+    = favicon_link_tag asset_path("images/govuk-apple-touch-icon-167x167.png"), rel: "apple-touch-icon", type: "image/png", size: "167x167" 
+    = favicon_link_tag asset_path("images/govuk-apple-touch-icon-180x180.png"), rel: "apple-touch-icon", type: "image/png", size: "180x180" 
+
+    = stylesheet_link_tag "application", "data-turbo-track": "reload" 
+    = javascript_include_tag "application", "data-turbo-track": "reload", defer: true
+
+  body class="govuk-template__body"
+    javascript: document.body.className = ((document.body.className) ? document.body.className + " js-enabled" : "js-enabled");
```

Comment:
One other other thing, sorry: Slim doesn't parse this correctly, so JS doesn't run correctly. To make it work you need a line break after `javascript:` and indent the next line 2 spaces further in than the `javascript:` indentation

Reply:
thanks, this was one I was concerned about